Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -262,7 +262,7 @@ - (void)nextTapped:(__unused id)sender {
 #pragma mark - STPFormTextFieldDelegate
 
 - (void)formTextFieldTextDidChange:(STPFormTextField *)textField {
-    if (self.type != STPAddressFieldTypeCountry) {
+    if (self.type != STPAddressFieldTypeCountry && self.type != STPAddressFieldTypeSEPACountry) {
```

Comment:
Hmm, is this a bug for regular cards also? It looks like everywhere but here the code assumes `self.contents` is the country code.